src/network/networkd-dhcp-static-lease.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#pragma once
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "conf-parser.h"
+
+#include "networkd-network.h"
+#include "networkd-util.h"
+
+
+typedef struct DHCPStaticLease DHCPStaticLease;
+typedef struct Network Network;
+typedef struct NetworkConfigSection NetworkConfigSection;
+
+struct DHCPStaticLease {
+        Network *network;
+        NetworkConfigSection *section;
+
+        struct in_addr address;
+        struct ether_addr client_id;
```

Comment:
why not just use an sd_dhcps_static_lease object for these two fields? You are going to convert them later on anyway, so why not store things like that right from the beginning